spec/query_objects/filtered_articles_query_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe FilteredArticlesQuery do
+  subject(:all) { query.all }
+
+  let(:query) { described_class.new(relation, filter_params) }
+  let(:relation) { Article.all }
+
+  context "with no params" do
+    let(:filter_params) { {} }
+
+    it { is_expected.to match_array(relation) }
+  end
+
+  describe "search param" do
```

Comment:
you can read about differences between `describe` & `context` here https://www.betterspecs.org/#describe
```suggestion
  context "with search params" do
```